src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractArmBatchProcessResponseFiles.java

```diff
@@ -328,6 +329,29 @@ && nonNull(armResponseBatchData.getUploadFileFilenameProcessor())) {
         );
     }
 
+    private void logResponsesFound(ArmResponseBatchData armResponseBatchData) {
+        log.info("Found ARM responses for external object directory ID {} with {} invalid line files, {} create record files and {} upload files",
+                 armResponseBatchData.getExternalObjectDirectoryId(),
+                 armResponseBatchData.getInvalidLineFileFilenameProcessors().size(),
+                 nonNull(armResponseBatchData.getCreateRecordFilenameProcessor()) ? 1 : 0,
+                 nonNull(armResponseBatchData.getUploadFileFilenameProcessor()) ? 1 : 0);
+        if (nonNull(armResponseBatchData.getCreateRecordFilenameProcessor())) {
+            log.info("Found eod {} with create record file: {}", armResponseBatchData.getExternalObjectDirectoryId(),
+                     armResponseBatchData.getCreateRecordFilenameProcessor().getCreateRecordFilenameAndPath());
+        }
+        if (nonNull(armResponseBatchData.getUploadFileFilenameProcessor())) {
+            log.info("Found eod {} with upload file: {}", armResponseBatchData.getExternalObjectDirectoryId(),
+                     armResponseBatchData.getUploadFileFilenameProcessor().getUploadFileFilenameAndPath());
+        }
+        if (CollectionUtils.isNotEmpty(armResponseBatchData.getInvalidLineFileFilenameProcessors())) {
+            armResponseBatchData.getInvalidLineFileFilenameProcessors().forEach(
+                invalidLineFileFilenameProcessor -> log.info("Found eod {} with invalid line file: {}",
+                                                             armResponseBatchData.getExternalObjectDirectoryId(),
+                                                             invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath())
+            );
+        }
+    }
```

Comment:
Could some of the `armResponseBatchData` attributes be assigned to variables, to avoid the repeated calls to their getters? I think it would help reduce the verbosity of the code somewhat.

Reply:
There is a ticket to refactor this class as its nearly 1000 lines long so might be best left for that?